### PR TITLE
[WIP] Add cli subcommand for component uninstall

### PR DESCRIPTION
**Pull request by ipochi**
This PR builds on #73 

Currently we redirect the user to use helm for uninstalling the component or piping the output of `lokoctl component render-manifest` to `kubectl delete -f` which isn't ideal.

This PR addresses the need to have a simple uninstallation command
`lokoctl component uninstall <component-name>`

### CODE REVIEW

File: cli/cmd/component-uninstall.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	pf.BoolVarP(&confirmUninstall, "confirm", "", false, "Conform component uninstall")
+}
+
+func runUninstall(cmd *cobra.Command, args []string) {
+	contextLogger := log.WithFields(log.Fields{
+		"command": "lokoctl component uninstall",
+		"args":    args,
+	})
+
+	lokoConfig, diags := getLokoConfig()
```
Comment (invidian) [inline cli/cmd/component-uninstall.go]: Why is this needed?

File: cli/cmd/component-uninstall.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	if diags.HasErrors() {
+		contextLogger.Fatal(diags)
+	}
+
+	var componentsToUninstall []string
+	if len(args) > 0 {
+		componentsToUninstall = append(componentsToUninstall, args...)
```
Comment (invidian) [inline cli/cmd/component-uninstall.go]: Why is this needed?

File: cli/cmd/component-uninstall.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	}
+}
+
+func uninstallComponents(lokoConfig *config.Config, kubeconfig string, componentsToUninstall ...string) error {
+	for _, componentName := range componentsToUninstall {
+		fmt.Printf("Uninstalling component '%s'...\n", componentName)
+
+		component, err := components.Get(componentName)
+		if err != nil {
+			return err
```
Comment (invidian) [inline cli/cmd/component-uninstall.go]: Please annotate errors to make tracing errors easier for the future (use `fmt.Errorf` and describe what happened).

File: cli/cmd/component-uninstall.go [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+		}
+
+		if err := helmutil.UninstallRelease(componentName, component, kubeconfig); err != nil {
```
Comment (invidian) [inline cli/cmd/component-uninstall.go]: Maybe `components` package should have `Uninstall` function, so then we also don't need to call `Get`?

**Comment by invidian** [comment]
ping @ipochi

**Comment by invidian** [comment]
Should we close this PR in favor of #268? CC @surajssd @ipochi

**Comment by iaguis** [comment]
This was implemented in #268. Closing.

**Comment by invidian** [comment]
@ipochi can we remove the branch too?
